dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBMongoDB/AzureCosmosDBMongoDBVectorStoreRecordCollection.cs

```diff
@@ -298,7 +301,7 @@ public async IAsyncEnumerable<VectorSearchResult<TRecord>> VectorizedSearchAsync
 
         // Constructing a query to fetch "offset + limit" total items
         // to perform offset logic locally, since offset parameter is not part of API. 
-        var itemsAmount = searchOptions.Offset + searchOptions.Limit;
+        var itemsAmount = searchOptions.Skip + searchOptions.Top;
```

Comment:
nit: the comment above still has offset and limit words.

Reply:
Fixed, thanks.